src/appkit.cs

```diff
@@ -16942,7 +16947,7 @@ public partial interface NSTextInputContext {
 		NSTextInputContext CurrentInputContext { get; }
 
 		[Export ("client")]
-		NSTextInputClient Client { get; }
+		INSTextInputClient Client { get; }
```

Comment:
that looks like a breaking change - unless the type is new and miss it's Mac(10,12) attribute ?